Views with galaxy custom strings and Styled Email Template

tatiaris opened the pull request:
Added custom strings to the verification page and signup page.
(were there other pages I had to do this for?)

phulsechinmay commented:
@tatiaris On mobile the email looks kinda weird:

<img width="414" alt="image" src="https://user-images.githubusercontent.com/22554368/84428293-d48e9600-abeb-11ea-9a17-c3641b451bb5.png">

Might need to fix this before merging

tatiaris commented:
Fixed the issues, can you review again? @phulsechinmay 
- fixed activate button positioning
- made the verification email sent page subtext hard coded instead of a galaxy custom string
